bindings/c/manifoldc.cpp

```diff
@@ -702,6 +703,20 @@ int manifold_get_circular_segments(double radius) {
 
 void manifold_reset_to_circular_defaults() { Quality::ResetToDefaults(); }
 
+ManifoldTriangulation *manifold_triangulate(void *mem, ManifoldPolygons *ps,
+                                            double epsilon) {
+  auto triangulation = manifold::Triangulate(*from_c(ps), epsilon);
+  return to_c(new (mem) std::vector<ivec3>(triangulation));
```

Comment:
Isn't `auto` already just `std::vector<ivec3>`?

Reply:
For in-place new I think you do need this to specify the type, it is not a type cast.